### ISSUE TITLE
select: improves with a11y

### ISSUE THREAD

**Issue by ThomasBurleson** [angular/material#9908]
-  The aria-label should not include the selected value.
-  The element for the placeholder value should have role=”option”, aria-posinset=”(relative_position_in_list)”, and aria-sizeof=”(number_of_items)”. The latter two ARIA attributes are to trick the screen reader into thinking that there is more than one option.
-  Drop the aria-owns.
-  Hide the place holder when the option list is shown.
-  Put role=”presentation” on the option list container.

**Comment by EladBezalel** [comment]
regarding `aria-posinset="(relative_position_in_list)"`- what about multi select?

**Comment by npbenjohnson** [comment]
This is also a problem in relation to #10737 because screen readers ignore aria-label if aria-labelledby is present. I have a different solution from the suggestion working locally. I just pointed aria-describedby at the summary container and added a hidden "Selected Value(s): " span. This makes md-select-text and md-select-html the information for screen readers too, which I would debate is probably the correct behavior.

**Comment by Splaktar** [comment]
Closing as many of these were fixed in 1.1.5 (https://github.com/angular/material/pull/10760) and others have open issues that specifically reference just one of these bullets.
